archetype-common/src/test/java/org/apache/maven/archetype/old/ArchetypeTest.java

```diff
@@ -453,5 +489,6 @@ protected void setUp()
     {
         super.setUp();
         archetype = (OldArchetype) lookup( OldArchetype.ROLE );
+        out = new StringWriter();
```

Comment:
setUp method usually goes near the top

Reply:
I kept the method where it was before my patch. I think that keeps reviewing changes easier.
While I personally also prefer it at the top, there is another class which has it after the test, so I assume this is somehow an intended style.